index.js

```diff
@@ -0,0 +1,179 @@
+// import { List } from "./modules/booklist";
+
+// const list = new List();
+// list.load();
+
+const section = document.querySelector('.container');
+const mainForm = document.getElementById('book-form');
+const displayPage = document.querySelector('.display-page');
+const currentDate = document.createElement('div');
+currentDate.id = 'displayDateTime';
+const navBar = document.createElement('ul');
+const navMenuDiv = document.createElement('div');
+navMenuDiv.className = 'nav-menu-div';
+navBar.className = 'nav-bar';
+const navHeading = document.createElement('h2');
+navHeading.textContent = 'Awesome Books';
+navHeading.className = 'nav-title';
+const listMenu = document.createElement('li');
+listMenu.id = 'first-list';
+const listLink = document.createElement('a');
+listLink.textContent = 'List';
+listLink.className = 'menu-link';
+listMenu.append(listLink);
+navMenuDiv.append(listMenu);
+
+const addMenu = document.createElement('li');
+addMenu.id = 'second-list';
+const addLink = document.createElement('a');
+addLink.textContent = 'Add Book';
+addLink.className = 'menu-link';
+addMenu.append(addLink);
+navMenuDiv.append(addMenu);
+
+const contactMenu = document.createElement('li');
+contactMenu.id = 'third-list';
+const contactLink = document.createElement('a');
+contactLink.textContent = 'Contact';
+contactLink.className = 'menu-link';
+contactMenu.append(contactLink);
+navMenuDiv.append(contactMenu);
+navBar.append(navMenuDiv);
+navBar.prepend(navHeading);
+
+const today = new Date();
+const date = `${today.getFullYear()}-${today.getMonth() + 1}-${today.getDate()}`;
+const x = today.getHours() < 12 ? 'am' : 'pm';
+const time = `${today.getHours()
+}:${
+  today.getMinutes()
+}:${
+  today.getSeconds()
+} ${
+  x}`;
+
+const dateTime = `${date} ${time}`;
```

Comment:
- [ ] I can see that you manually implemented current date and time functionality, this is very impressive, good job 👏🏻 . 
But I will have to ask you to use `luxon` library to render current time, as it is an essential part of current milestone (( 

I can also see that you tried to add luxon, but for some reason it did not work, please go through the [documentation](https://moment.github.io/luxon/#/) and you can see getting started steps. 

You can also reference my project where me and my coding partner were implementing `luxon` https://github.com/Timowest12/Awesome-books/pull/5/files